query/src/main/scala/filodb/query/exec/rangefn/RangeFunction.scala

```diff
@@ -298,18 +298,19 @@ object RangeFunction {
                           func: Option[InternalRangeFunction],
                           funcParams: Seq[Any] = Nil): RangeFunctionGenerator = {
     func match {
-      case None                 => () => new LastSampleChunkedFunctionL
-      case Some(CountOverTime)  => () => new CountOverTimeChunkedFunction()
-      case Some(SumOverTime)    => () => new SumOverTimeChunkedFunctionL
+      case None                   => () => new LastSampleChunkedFunctionL
+      case Some(CountOverTime)    => () => new CountOverTimeChunkedFunction()
```

Comment:
Why are these moved?

Reply:
Because I was asked to align all of them